src/api/PDFPage.ts

```diff
@@ -661,6 +663,46 @@ export default class PDFPage {
     );
   }
 
+  /**
+   * Draw a line on this page. For example:
+   * ```js
+   * import { degrees, grayscale, rgb } from 'pdf-lib'
```

Comment:
Can we remove `degrees` and `grayscale` from the import list, since they aren't used in this example?

Reply:
OK

Reply:
hi @Hopding ,

I did the requested changes and added a call to 'drawLine' in 'test1' integration test for both 'node' and 'web' but when I try to compile the project 'yarn build' or even verify the types with 'yarn typecheck', I am getting the following errors:

$ yarn typecheck
yarn run v1.17.3

30 declare var beforeEach: jest.Lifecycle;
               ~~~~~~~~~~
    'beforeEach' was also declared here.

node_modules/@types/jest/index.d.ts:31:13 - error TS2300: Duplicate identifier 'afterAll'.

31 declare var afterAll: jest.Lifecycle;
               ~~~~~~~~
    'afterAll' was also declared here.

node_modules/@types/jest/index.d.ts:32:13 - error TS2300: Duplicate identifier 'afterEach'.

32 declare var afterEach: jest.Lifecycle;
               ~~~~~~~~~
    'afterEach' was also declared here.

node_modules/@types/jest/index.d.ts:36:13 - error TS2300: Duplicate identifier 'it'.

36 declare var it: jest.It;
               ~~
    'it' was also declared here.

node_modules/@types/jest/index.d.ts:37:13 - error TS2300: Duplicate identifier 'fit'.

37 declare var fit: jest.It;
               ~~~
    'fit' was also declared here.

node_modules/@types/jest/index.d.ts:38:13 - error TS2300: Duplicate identifier 'xit'.

38 declare var xit: jest.It;
               ~~~
    'xit' was also declared here.

node_modules/@types/jest/index.d.ts:1077:46 - error TS2314: Generic type 'ArrayContaining<T>' requires 1 type argument(s).

1077     function arrayContaining(sample: any[]): ArrayContaining;
                                                  ~~~~~~~~~~~~~~~
node_modules/@types/jest/index.d.ts:1078:45 - error TS2314: Generic type 'ObjectContaining<T>' requires 1 type argument(s).

1078     function objectContaining(sample: any): ObjectContaining;
                                                 ~~~~~~~~~~~~~~~~
node_modules/@types/jest/index.d.ts:1104:15 - error TS2428: All declarations of 'ArrayContaining' must have identical type parameters.

1104     interface ArrayContaining {
                   ~~~~~~~~~~~~~~~
node_modules/@types/jest/index.d.ts:1110:15 - error TS2428: All declarations of 'ObjectContaining' must have identical type parameters.

1110     interface ObjectContaining {
                   ~~~~~~~~~~~~~~~~
node_modules/@types/jest/index.d.ts:1238:9 - error TS2687: All declarations of 'message' must have identical modifiers.

1238         message: string | (() => string);
             ~~~~~~~
Found 13 errors.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Could you please help to solve these issues?

Note: When I first cloned the project, I did both commands (build and typecheck) and both worked fine.

Regards,

Reply:
Hello @samezyane. Sorry for the late response. It sounds like the dependencies got messed up somehow. Have you tried running the following to reinstall them?
```
$ rm -rf node_modules
$ yarn install
```